pages/index.tsx

```diff
@@ -46,64 +47,84 @@ interface IHomeProps {
 export const getServerSideProps:GetServerSideProps = async ({ req }) => {
 
 	const network = getNetworkFromReqHeaders(req.headers);
-	if(isOpenGovSupported(network) && !req.headers.referer) {
-		return {
-			props: {},
-			redirect: {
-				destination: '/opengov'
-			}
-		};
-	}
+	// if(isOpenGovSupported(network) && !req.headers.referer) {
```

Comment:
comments here too

Reply:
left these comments intentionally in case i need to revert the changes , will remove all the commented code

Reply:
removed